raster-foundry/raster-foundry
# Fix Baseline Flyway Migration and AWS Access for GDAL

## Overview

This is two minor fixes:
 - Updates the baseline migration (which shouldn't be applied anywhere yet) to include the schema of metrics table updated
 - Uses `AWS_DEFAULT_PROFILE` for settings AWS profile to enable usage of GDAL properly

### Checklist


### Testing Instructions

- Pull down and rebuild servers
- Run `./scripts/load_development_data --download` and get the newest version of the development database
- Run `./scripts/migrate info` and verify it does not error
- Start server + frontend - make sure the project page loads all projects
